### PR TITLE
New fast optical simulation trained with dune-vd v5 geometry

### PR DESCRIPTION
**Pull request by Li-Jiaoyang97**
New parameterisations for semi-analytical model trained with dune-vd v5 geometry, together with new libraries trained with v5 geometry. This PR goes with [larsim PR #118](https://github.com/LArSoft/larsim/pull/118) which has already been merged into the larsim v09_37_00 release for larsoft v09_76_00 release.

### CODE REVIEW

**Review by pgreen135 — APPROVED** [review]

File: duneopdet/PhotonPropagation/PDFastSim_dune.fcl [hunk 1]
```diff
@@ -72,8 +72,8 @@ dunevd_pdfastsim_par_ar_fastonly.DoSlowComponent: false
 dunevd_pdfastsim_par_xe:                       @local::standard_pdfastsim_par_ar
 dunevd_pdfastsim_par_xe.DoFastComponent:       false          # Only slow Xe light
 dunevd_pdfastsim_par_xe.VUVHits:               @local::dunevd_vuv_Xenon_hits_parameterization
-dunevd_pdfastsim_par_xe.VISHits:               @local::dunevd_vuv_Xenon_anode_hits_parameteriszation
-dunevd_pdfastsim_par_xe.IncludeAnodeReflections: true		  # include Xenon light reflections from anode
+# dunevd_pdfastsim_par_xe.VISHits:               @local::dunevd_vuv_Xenon_anode_hits_parameteriszation
+dunevd_pdfastsim_par_xe.IncludeAnodeReflections: false		  # include Xenon light reflections from anode
```
Comment (lpaulucc) [inline duneopdet/PhotonPropagation/PDFastSim_dune.fcl:76]: Why is this now set to false?

File: duneopdet/PhotonPropagation/PDFastSim_dune.fcl [hunk 2]
```diff
@@ -115,10 +115,10 @@ protodune_hd_pdfastsim_pvs_external:                          @local::standard_p
 protodune_hd_pdfastsim_pvs_external.OnlyActiveVolume: 	      false
 protodune_hd_pdfastsim_pvs_external.SimulationLabel:          IonAndScintExternal
 protodune_hd_pdfastsim_pvs_external.IncludePropTime:  	      true   
-protodune_hd_pdfastsim_pvs_external.DoFastComponent:    	  true
-protodune_hd_pdfastsim_pvs_external.DoSlowComponent:    	  false
-protodune_hd_pdfastsim_pvs_external.DoReflectedLight:   	  false         
-protodune_hd_pdfastsim_pvs_external.VUVTiming:	  	          @local::dune_vuv_timing_parameterization
+protodune_hd_pdfastsim_pvs_external.DoFastComponent:          true
+protodune_hd_pdfastsim_pvs_external.DoSlowComponent:          false
```
Comment (lpaulucc) [inline duneopdet/PhotonPropagation/PDFastSim_dune.fcl:119]: Why false here if it uses the argon modelling?

File: duneopdet/PhotonPropagation/photpropservices_dune.fcl [hunk 3]
```diff
@@ -268,6 +268,54 @@ dune10kt_vd_photonvisibilityservice_ArXe: # Generated using 1x8x14 geometry.
...
+  NY: 174
+  NZ: 220
+
+  UseCryoBoundary: false
+  DoNotLoadLibrary: false
+  LibraryBuildJob: false
+  # IncludePropTime: true
+
+  LibraryFile: "PhotonPropagation/LibraryData/PhotonLibrary_dunevd10kt_3view_30deg_v5_refactored_1x8x14ref_argon_active_removed.root"    
+  # This is the merged library between argon and xenon, for the hybrid model.
```
Comment (lpaulucc) [inline duneopdet/PhotonPropagation/photpropservices_dune.fcl:284]: If this is the merged library, why do we have one configuration for argon and another one for xenon?

File: duneopdet/PhotonPropagation/PDFastSim_dune.fcl (hunk 1 above)
Comment (Li-Jiaoyang97) [reply duneopdet/PhotonPropagation/PDFastSim_dune.fcl:76]: Hi @lpaulucc, this was due to the fact the anode reflection is on 12% now thus we were comparing the old case where we were treating anode as a separate reflection and included other reflections (field cage, other part of the detector) as border effects, with the other case where we incorporate everything (including anode reflection) as border effects. Sidenote: the second one is what we have already been used to for the argon case. And the comparisons show that the latter case have a better resolution when compared to GEANT4. Thus we move forward and decided to go with the latter case.

Comment (lpaulucc) [reply duneopdet/PhotonPropagation/PDFastSim_dune.fcl:76]: Thank you for the explanation

**Comment by lpaulucc** [comment]
Let's hold this merging until we have a clear picture of the argon/xenon library situation and why it seems the slow argon component is not accounted for.

**Comment by tomjunk** [comment]
There's a 2-week-old comment saying to old the merging until there's a clear picture of the library situation, but it has been added to the merge milestone.  Shall we convert this PR to a draft so we don't accidentally merge it?  Or go ahead and merge it?

**Comment by lpaulucc** [comment]
Hi Tom, I spoke to Jiaoyang today (she was on holiday, therefore the two week gap) and she will update this PR tomorrow morning. After that, it will be ready to merge.

**Comment by Li-Jiaoyang97** [comment]
Hi @lpaulucc and @tomjunk. Yeah I was on holidays and thus why the PR is two-week old. I think I have finished all corrections that Laura wants me to change. :))

**Pull request merged**